### PR TITLE
Fix vnet creation to accept dns servers

### PR DESCRIPTION
**Pull request by katcipis**
Small fix, on another MR will update deps so I can check if DNS servers are being properly configured.

### CODE REVIEW

File: azure/vnet.sh [hunk 1]
```diff
@@ -1,7 +1,18 @@
 # Virtual Network related functions
 
-fn azure_vnet_create(name, group, location, cidr) {
-	azure network vnet create --name $name --resource-group $group --location $location --address-prefixes $cidr
+fn azure_vnet_create(name, group, location, cidr, dnsservers) {
+        fn join(list, sep) {
```
Comment (katcipis) [inline azure/vnet.sh]: later we could use the stdlib here :-)

Comment (tiago4orion) [inline azure/vnet.sh]: :+1:

**Comment by tiago4orion** [comment]
We need to freeze the azure cli version we use in the Makefile to avoid this kind of bug...
And... if #25 were fixed we would get this bug very earlier :-)

**Comment by katcipis** [comment]
@tiago4orion could not agree more, let me just finish the basic stuff and I will take a look on these :-)

File: azure/vnet.sh (hunk 1 above)
Comment (kamilash) [inline azure/vnet.sh]: It seems that this func could be reused by others.

**Comment by katcipis** [comment]
@kamilash we plan on adding a join on nash stdlib, for now it is just used there so I did not find any reason to start a join module inside klb

**Pull request merged**